repositories.bzl

```diff
@@ -549,9 +549,9 @@ cc_proto_library(
         name = "googleapis_git",
         build_file_content = BUILD,
         patch_cmds = ["find . -type f -name '*BUILD*' | xargs rm"],
-        strip_prefix = "googleapis-ae7a4cc69cc1e206b16f1b9db803907d7a3d97c8",  # Oct 22, 2019
-        url = "https://github.com/googleapis/googleapis/archive/ae7a4cc69cc1e206b16f1b9db803907d7a3d97c8.tar.gz",
-        sha256 = "f96e11515c302045e8ab6708ba68d7cea8a02e2a96add92033315ff894076980",
+        strip_prefix = "googleapis-11fd1fb53d03cf4852d3f63d679292e3c7b6e7c8",  # Oct 21, 2020
```

Comment:
Updating is good. But I'm curious, why do you need `google.rpc.ErrorInfo` in the build? Is there some error you are seeing?

Reply:
Yes, we get an error. When we tried to add it to `google.rpc.Status` - field `details` (using `com.google.protobuf.Any.pack()` in a Java server), ESP returns "Invalid type URL, unknown type: google.rpc.ErrorInfo".

We tried to include an updated version of `google/rpc/error_details.proto` to our service definition, but for some reason it seemed to be filtered out. That could be caused by the Maven plugin (`org.xolstice.maven.plugins:protobuf-maven-plugin:0.6.1`) we use to generate `descriptor.pb`, as I could not see any `google.rpc` messages included when parsing it with `protoc`.

Reply:
The right way to do is:
1) make sure your descriptor.pb has the proto message: google.rpc.ErrorInfo. Then pass this descriptor.db to "gcloud endpoints services deploy"
2) in the endpoints service yaml file, please add "Types" [field](https://github.com/googleapis/googleapis/blob/master/google/api/service.proto#L113) to specify all "Any" types you used in your grpc server.   Here is an [example](https://github.com/cloudendpoints/esp/blob/master/test/transcoding/bookstore-api.yaml#L31) on how to specify it. 

After this, ESP will know how to translate these proto messages into JSON.

Reply:
Thank you very much @qiwzhang ! I was not aware of step 2 - that explains why using a custom `ErrorInfo` (which _did_ show up in `descriptor.pb`) didn't work for us either.

However, for unknown reason, we didn't manage to add `google.rpc.ErrorInfo` to `descriptor.pb`, which this PR would work around by including it in ESP.

Reply:
Yup, updating it here will not solve your problem. Wayne described it well. 

Basically `google.rpc.Status.details` [is a google.protobuf.Any](https://github.com/googleapis/googleapis/blob/0f626de4544db840d04b9cd777dfb1197702244a/google/rpc/status.proto#L46). ESP does not handle these Any automatically, so the `google.rpc.ErrorInfo` must be added to `types`.

Side note: ESPv2 does not have this problem, it works automatically.

Reply:
It will not since ESP is NOT using that proto message.   Just like any c++ binary,  if a binary is using a proto message, its type will be in the global descriptorset.   ESP is using the global descriptor set + the types from Endpoint service config to do the transcoding.

Reply:
@oyvindwe I don't think adding it to ESP will add it to your descriptor. The descriptor is specific for your Cloud Endpoints service, I doubt it will be merged with the protos ESP imports.

One simple way to make sure it shows up in your descriptor is to add a dummy unused message that has `google.rpc.ErrorInfo`.

```
message UnusedMessage {
   google.rpc.ErrorInfo error_info = 1;
}
```

@qiwzhang to confirm

Reply:
Are you guys sure about that updating will not solve it? We successfully expose `DebugInfo` and `LocalizedMessage` in this way, and we have not added any to `types`.

Reply:
Btw, I am very grateful for the example above, and if we don't get it to work with `google.rpc.ErrorInfo`, we can just use a custom message type.

Good to know that this should not be a problem when we upgrade to ESPv2 as well - thanks for the info!